conda_press/condatools.xsh

```diff
@@ -687,8 +694,27 @@ class ArtifactInfo:
                 dep.clean()
 
 
+def get_only_deps_on_pypi(list_deps):
+    """Based on a list of dependencies this function will check if those
+    dependencies are on PyPi, if it is not available it will be removed.
+
+    :param set,list list_deps: List of dependencies as a string values
+    :return set: List of pakages present on PyPi
+    """
+    import requests
+    new_deps = set()
+    for pkg in list_deps:
+        response = requests.get(f"https://pypi.python.org/pypi/{pkg.lower()}/json")
```

Comment:
I think PyPI also replaces `-` with `_` in package names.

Reply:
Indeed! Good catch, I completely forgot about it!
I am going to modify this part and I will also create some tests to cover this scenario. Thanks!

Reply:
I was testing it, and it seems that ``pypi`` now accept the link to be with ``-`` or ``_``.
for example, with ``pytest-xdist``
https://pypi.org/pypi/pytest-xdist/json
https://pypi.org/pypi/pytest_xdist/json

both links return the metadata of ``pytest-xdist``

Reply:
But I will replace the ``-`` by ``_``